npm_and_yarn/lib/dependabot/npm_and_yarn.rb

```diff
@@ -240,6 +254,12 @@ def self.sanitize_resolvability_message(error_message, dependencies, yarn_lock)
           Dependabot::MisconfiguredTooling.new("Yarn", message)
         }
       },
+      "YN0060" => {
+        message: "Incompatible peer dependency",
```

Comment:
I think :dependabot: shouldbe handling this class of errors

Reply:
@abdulapopoola , we can force resolve using _--force_ or user can manually resolve the incompatibility. i wont recommend _--force_ , i'm removing the handler for now and will look as part of  [this ongoing issue](https://github.com/dependabot/dependabot-core/tree/ssandhu/dependabot-shared-helpers-err-pnpm-peer-dep-issues) .